nucliadb_node/Cargo.toml

```diff
@@ -110,9 +110,6 @@ tonic-build = "0.11.0"
 ci-test = []
 release-feature-set = []
 
-[profile.release]
-lto = true
-
```

Comment:
This was being ignored, so I removed it.
I tried adding it back to `.cargo/config.toml` where it wouldn't be ignored, but it was VERY slow (I got bored after 3 minutes, without it takes 5 seconds or so), so removing it for now.